archetypes/pattern/_index.adoc

```diff
@@ -0,0 +1,32 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
+summary:
+products:
+- Product 1
+- Product 2
+- etc
+industries:
+- Industry 1
+- Industry 2
+- etc
+links:
+  install:
+  arch:
+  help: https://groups.google.com/g/hybrid-cloud-patterns
+  bugs:
+# validated:
+# pattern_logo:
+# ci:
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+
+// Add asciidoc modules here
```

Comment:
Can we add a suggestion about using the [include::](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to add modules? 
For the other archetypes, such as blog or learn, we suggest "_add you content/blog here_ ", which is fairly straightforward. However, it might help to clarify that for the patterns pages, one must create modules and add them by using the `include` directive.